kuma/javascript/src/article.jsx

```diff
@@ -47,6 +48,13 @@ export default function Article({ document }: DocumentProps) {
 
     const isArchive =
         document.slug === 'Archive' || document.slug.startsWith('Archive/');
+    const locale = getLocale();
+
+    useEffect(() => {
+        if (document.locale !== locale) {
+            mdn.analytics.trackError('Translation Pending', 'displayed');
```

Comment:
I wonder if we should not instead use the `ga` functionality in the new frontend? One reason is that this will most likely fail when server-side rendered as `mdn.analytics` will not be defined.

Looking at the code in `/kuma/static/js/analytics.js` the `trackError` function calls `trackEvent` and passes the following in addition to what is passed into the function:

```
{
  'category': category, // 'Translation Pending'
  'action': action, // 'displayed'
  'label': label // ''
}
```

`trackEvent` calls `send` which then does the final call:

```
ga('send', {
  'eventCategory' : 'Translation Pending',
  'eventAction'   : 'displayed',
  'eventLabel'    : '',
  'eventValue'    : 0,
  'hitCallback'   : null,
  'hitType': 'event'
});
```

So for this we can do:

```
...
import GAProvider from './ga-provider.jsx';
...
const ga = useContext(GAProvider.context);
...
ga('send', {
  hitType: 'event',
  eventCategory: 'Translation Pending',
  eventAction: 'displayed',
  eventLabel: ''
});
```

Reply:
You're right @schalkneethling that `const ga = useContext(GAProvider.context);` is the new better API. 

However, in its current state `mdn.analytics.trackError` is only ever reached for from an effect (`useEffect`) and they only execute when in the browser.

Reply:
Ah yes, that is true. But, would it not seem a little out of place to have this one place where we can into the "old" JS codebase, but everywhere else we use the new frontend GA bits?

Reply:
THANK YOU @schalkneethling for that reminder. I put this up: https://github.com/escattone/kuma/pull/2
@escattone ping ^

Reply:
Merged. Thanks @schalkneethling and @peterbe!